src/anonymous-credentials/credential-builder.ts

```diff
@@ -171,6 +166,23 @@ export class CredentialBuilder extends Versioned {
     if (this._credStatus !== undefined) {
       s[STATUS_STR] = this._credStatus;
     }
+
+    let schema = this.schema as CredentialSchema;
```

Comment:
Why is this moved here? Previously only the sign function had to know about signing options, now serialization function also has to know about it.

Reply:
Here https://github.com/docknetwork/crypto-wasm-ts/pull/12/commits/ce3459186d444e31b694c6acf4837c7bc6196835